README.md

```diff
@@ -141,18 +142,18 @@ build/Registry.json: "address": "0xf25186B5081Ff5cE73482AD761DB0eB0d25abfBF",
 ```
 
 From the output shown above you need to look for the contract address shown.
-As a result the registry address in the example above is `0xf25186B5081Ff5cE73482AD761DB0eB0d25abfBF`
+As a result the registry address in the example above is `0xf25186B5081Ff5cE73482AD761DB0eB0d25abfBF`.
 
 
 ## Deploy the "Hello World" Insurance Product
 
-Ensure that the DEV_GIF_REGISTRY variable in the dotenv file contains the right GIF registry address of the GIF instance where you want to deploy the "Hello World" insurance product.
+Ensure that the DEV_GIF_REGISTRY variable in the `dotenv` file contains the right GIF registry address of the GIF instance where you want to deploy the "Hello World" insurance product.
```

Comment:
if dotenv file should be used in quotes i'd rather use '.env' file instead

Reply:
sounds good